crates/svm-sdk/storage/src/mock.rs

```diff
@@ -173,7 +173,7 @@ fn storage() -> MutexGuard<'static, InnerStorage> {
 mod tests {
     use super::*;
 
-    use crate::memory::alloc;
+    use svm_sdk_alloc::alloc;
```

Comment:
`STORAGE` instance init/access pattern is different than `HOST` - If there's no reason for that, it should be unified.